swatch-core/schemas/event.yaml

```diff
@@ -0,0 +1,153 @@
+$schema: http://json-schema.org/draft-07/schema#
```

Comment:
This should have been flagged as a move by git. I'm thinking you need to amend... something like `git rm -rf schemas; git commit --amend` should work